spk/sonarr/Makefile

```diff
@@ -20,7 +18,7 @@ DESCRIPTION_SPN = Sonarr es un PVR para los usuarios de grupos de noticias. Se p
 ADMIN_PORT = 8989
 RELOAD_UI = yes
 DISPLAY_NAME = Sonarr
-CHANGELOG = "Now runs mono in debug mode"
+CHANGELOG = "1. No longer in beta"
```

Comment:
I'm sorry for the nitpicking, but for one entry, the `1.` wouldn't be necessary :)